param/low_level_motion_0.05.yaml

```diff
@@ -81,3 +81,6 @@ brace_impact_failure_height: 0.2
 
 # This parameter will be obsolete when the foot touch sensors are implemented
 brace_impact_success_height: 0.05
+
+#maximum height at which the landing gear switches should be toggled
+switch_toggle_height: 0.04
```

Comment:
Can we reword this to, "height at which the contact switches must be unpressed" and rename the param to takeoff_max_height_switch_pressed? The variable in the class can be named max_height_switch_pressed.

Also can we raise it to 0.25. 0.04 is really low